docs/src/main/tut/docs/README.md

```diff
@@ -76,44 +95,21 @@ Once you have these abstract definitions, you can combine them in whichever way
 
 Learn more about [modules](./core/modules) in the extended documentation.
 
-## Building programs
-
-Abstract definitions are all it takes to start building programs that support sequential and parallel operations that are entirely decoupled from their runtime interpretation.
-
-The example below combines both algebras to produce a more complex program:
-
-```tut:book
-def program[F[_]](implicit A: Application[F]) = {
-  import A._
-  import cats.implicits._
-
-  for {
-    userInput <- interaction.ask("Give me something with at least 3 chars and a number on it")
-    valid <- (validation.minSize(userInput, 3) |@| validation.hasNumber(userInput)).map(_ && _).freeS
-    _ <- if (valid)
-            interaction.tell("awesomesauce!")
-         else
-            interaction.tell(s"$userInput is not valid")
-  } yield ()
-}
-```
-
 ## Running programs
 
 In order to run programs, we need interpreters. We define interpreters providing implementations for the operations defined in our algebras:
```

Comment:
one question here... is this supposed to be saying that we're defining interpreters that are providing implementations .... or,      we define interpreters by providing .... ??

Reply:
The second, we define interpreter and inside we provide the implementations